aiida/cmdline/commands/cmd_group.py

```diff
@@ -84,6 +84,70 @@ def group_remove_nodes(group, nodes, clear, force):
         group.remove_nodes(nodes)
 
 
+@verdi_group.command('move-nodes')
+@options.SOURCE_GROUP(required=True)
+@options.TARGET_GROUP(required=True)
+@arguments.NODES()
```

Comment:
This needs a help string. Since arguments can't have one, you should add it manually to the docstring of the command.